test/byContract/IexecEscrow/IexecEscrowToken.test.ts

```diff
@@ -0,0 +1,457 @@
+// SPDX-FileCopyrightText: 2024 IEXEC BLOCKCHAIN TECH <contact@iex.ec>
+// SPDX-License-Identifier: Apache-2.0
+
+import { AddressZero } from '@ethersproject/constants';
+import { loadFixture } from '@nomicfoundation/hardhat-network-helpers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { BigNumber } from 'ethers';
+import { ethers, expect } from 'hardhat';
+import { loadHardhatFixtureDeployment } from '../../../scripts/hardhat-fixture-deployer';
+import {
+    IexecInterfaceToken,
+    IexecInterfaceToken__factory,
+    RLC,
+    RLC__factory,
+} from '../../../typechain';
+import { getIexecAccounts } from '../../../utils/poco-tools';
+
+const standardAmount = AmountWithDecimals(BigNumber.from(100));
+const depositParams = {
+    amount: standardAmount,
+};
+const depositArgs = Object.values(depositParams) as [BigNumber];
+const withdrawParams = {
+    amount: standardAmount,
+};
+const withdrawArgs = Object.values(withdrawParams) as [BigNumber];
+
+describe('IexecEscrowTokenDelegate', () => {
+    let proxyAddress: string;
+    let [iexecPoco, , iexecPocoAsAccountA, iexecPocoAsAdmin]: IexecInterfaceToken[] = [];
+    let [iexecAdmin, accountA, accountB, anyone]: SignerWithAddress[] = [];
+    let [rlcInstance, rlcInstanceAsAccountA]: RLC[] = [];
+
+    beforeEach('Deploy', async () => {
+        proxyAddress = await loadHardhatFixtureDeployment();
+        await loadFixture(initFixture);
+    });
+
+    async function initFixture() {
+        const accounts = await getIexecAccounts();
+        ({ iexecAdmin, anyone: accountA, requester: accountB, anyone } = accounts);
+
+        iexecPoco = IexecInterfaceToken__factory.connect(proxyAddress, anyone);
+        iexecPocoAsAccountA = iexecPoco.connect(accountA);
+        iexecPocoAsAdmin = iexecPoco.connect(iexecAdmin);
+        rlcInstance = await RLC__factory.connect(await iexecPoco.token(), anyone);
+        rlcInstanceAsAccountA = rlcInstance.connect(accountA);
+        await rlcInstance
+            .connect(iexecAdmin)
+            .transfer(accountA.address, AmountWithDecimals(BigNumber.from(10_000)))
```

Comment:
```suggestion
            .transfer(accountA.address, standardAmount)
```
Would that be enough?

Reply:
Not enough for desposit for arrays but changed to `standardAmount.mul(10)` for consistencies